Automattic-Tracks-iOS/Crash Logging/SentryClient+Shim.m

```diff
@@ -0,0 +1,39 @@
+#import "SentryClient+Shim.h"
+
+@interface SentryClient(Internal)
+- (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event
+                             withScope:(SentryScope *)scope
+                alwaysAttachStacktrace:(BOOL)alwaysAttachStacktrace
+                          isCrashEvent:(BOOL)isCrashEvent;
+@end
+
+@implementation SentryClient(Shim)
+
+/*
+ * A wrapper around a private method in `SentryClient`. If it's unavailable (say the library changes),
+ * the worst that should happen is that the stack trace is no longer available on events sent manually using this method.
+ *
+ * We can remove this once the Sentry SDK allows for capturing events and being notified once they're delivered.
```

Comment:
Is there a GitHub issue on their end we can link here?

Reply:
Yeah, for sure – added in c6a751b926031609d8fe5eb49cbc2aac85589ddc